.eslintrc.js

```diff
@@ -0,0 +1,89 @@
+module.exports = {
```

Comment:
Just trying to understand what was done here. Were these rules pulled from airbnb-base?

Reply:
This is my mistake, I now remember you telling me to use the same config from sip-hosted-api with airbnb-base, I added ES6 rules for linting in this pull request. Ill make changes now to this and we can go through it again and hopefully all be correct